backend/nomad_region.go

```diff
@@ -246,10 +246,16 @@ func (n *NomadRegion) watchJobs() {
 
 func (n *NomadRegion) updateJob(job *api.Job) (*Action, error) {
 	if n.Config.NomadReadOnly {
-		logger.Errorf("Unable to run jon: NomadReadOnly is set to true")
+		logger.Errorf("Unable to run job: NomadReadOnly is set to true")
 		return &Action{Type: errorNotification, Payload: "The backend server is set to read-only"}, errors.New("Nomad is in read-only mode")
 	}
 
+	if n.Config.NomadHideEnvData {
```

Comment:
we actually don't need it there, this is only used internally, and not for user-facing submitting of a job - so the check can be removed... currently this is only called `changeTaskGroupCount`, which read the job from the API and submit it back, without ever showing it to a user

otherwise, the PR is good to merge!